spec/unit/reports/store_spec.rb

```diff
@@ -35,6 +35,30 @@
       Puppet::FileSystem.expects(:exist?).never
       expect { @report.process }.to raise_error(ArgumentError, /Invalid node/)
     end
+
+    context "when storing the report" do
+      context "and on Windows" do
+        before :each do
+          Puppet::Util::Platform.stubs(:windows?).returns(true)
+        end
+
+        it "does not set mode" do
+          Puppet::Util.expects(:replace_file).with() { |path, mode| mode.nil? }
+          @report.process
```

Comment:
nit, since `path` is unused, can you replace with `_` or `_path` so that it doesn't fail when merged up to later branches and cause problems when we run the `ruby -wc` check?

Reply:
Done